DynamicFeatures/features/assets/build.gradle

```diff
@@ -21,12 +21,16 @@ android {
     compileSdkVersion versions.compileSdk
 
     defaultConfig {
+
```

Comment:
why the empty lines after `{`, is it autoformatted like this?

Reply:
Reverted